sdk/src/Services/S3/Custom/Transfer/Internal/DownloadDirectoryCommand.cs

```diff
@@ -102,6 +102,10 @@ private TransferUtilityDownloadRequest ConstructTransferUtilityDownloadRequest(S
             downloadRequest.ServerSideEncryptionCustomerProvidedKey = this._request.ServerSideEncryptionCustomerProvidedKey;
             downloadRequest.ServerSideEncryptionCustomerProvidedKeyMD5 = this._request.ServerSideEncryptionCustomerProvidedKeyMD5;
             downloadRequest.RequestPayer = this._request.RequestPayer;
+            downloadRequest.ExpectedBucketOwner = this._request.ExpectedBucketOwner;
```

Comment:
i dont know if it matters here or now. but in the basecommand one i see we check `isSetExpectedBucketOwner` and similar before setting these values. wondering if it makes a difference or not. just something to think about, i dont know off the top of my head